### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by haliljon**
## In this PR I was able to:

> - Install Ruby on Rails framework.
> - Use Ruby Gems as a software packages system.
> - Understand what ORM is.
> - Use database migration files to maintain database schema.

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @haliljon,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a descriptive readme file. Kindly use the template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: config/database.yml [hunk 1]
```diff
@@ -57,6 +59,8 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
+  username: postgres
+  password: 8?rSjWDd#J5!a4T
```
Comment (brytebee) [inline config/database.yml:63]: - [ ] **Optional:** These credentials can be added to the default block so you don't need to repeat them in the dev, test and production blocks.

**Review by brytebee — APPROVED** [review]
Hi, @haliljon👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great DB design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/database.yml (hunk 1 above)
Comment (haliljon) [reply config/database.yml:63]: Hi @brytebee, thank you 🙏

**Pull request merged**